randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1097,7 +1105,13 @@ Extra - asset_id: collision_camera_010
   > Door to EMMI Zone Exit Southwest (DoorFrame_005)
       Trivial
   > Door to Wide Beam Block Room (DoorFrame_007)
-      Simple IBJ or Use Spin Boost
+      Speed Booster or Simple IBJ or Use Spin Boost
+  > Pickup (Power Bomb Tank)
+      All of the following:
+          Morph Ball and Speed Booster
```

Comment:
There is a Ballspark template that can be used here for slightly less clutter

Reply:
This technically isn't a ballspark though, it's a spark, and then a morph requirement to fit through the 1 block tunnel.

Reply:
Ballspark template is just if morph and speed are required, not necessarily meaning you have to ballspark